### ISSUE TITLE
Nice to have: Cargo support

### ISSUE THREAD

**Issue by jjpe** [p-e-w/ternimal#11]
I've seen in the README that Cargo isn't needed to build.  However, I'm finding that this increases the number of steps needed to install this project relative to having a Cargo.toml file:

With Cargo support, I could install ternimal with a one-liner:  `cargo install ternimal --git https://github.com/p-e-w/ternimal.git`.
This would download, build, and install ternimal in `~/.cargo/bin`, which is in `$PATH`.

However, atm I have to manually clone, then run `rustc`, and then copy the output manually to somewhere in `$PATH`.

In short, having Cargo support would be a very nice to have.

**Comment by p-e-w** [comment]
I will not be adding Cargo support. This is an aesthetic choice as discussed before in #3.
